CONTRIBUTING.md

```diff
@@ -87,20 +87,17 @@ We may also add references to JIRA tickets created on our side for the purpose o
 
 ### [Label Studio](https://github.com/humansignal/label-studio)
 
-The primary repository for the Label Studio Backend contains the majority of the logic for how labels are managed.
+The primary repository for Label Studio, and contains the majority of the logic for how labels are managed. Three areas where you may want to contribute include:
 
-### [Label Studio Frontend](https://github.com/HumanSignal/label-studio-frontend)
+* `label_studio`--This is the main app, containing most of the backend code. It also acts as the central integration point for all frontend elements. 
```

Comment:
Now `label_studio` folder keeps containing most of the backend code. However, the integration point for all frontend is on `web/apps/labelstudio` the same way that we changed lsf and dm from submodules to `web/libs` we moved the LSO frontend from `label_studio/frontend` to `web/apps`

Reply:
Updated as follows:

* `label_studio`--This is the main app, containing most of the backend code. 
* `web/apps/labelstudio` -- This acts as the central integration point for all frontend elements. 
* `web/libs/editor`--This is the frontend library. It uses React to build the UI and mobx-state-tree for state management. 
* `web/libs/datamanager`--This is the frontend interface for the Label Studio Data Manager, our data exploration tool.